clipper_admin_v2/clipper_admin/docker/docker_container_manager.py

```diff
@@ -0,0 +1,242 @@
+from __future__ import absolute_import, division, print_function
+import docker
+import logging
+import os
+from ..container_manager import (
+    ContainerManager, CLIPPER_DOCKER_LABEL, CLIPPER_MODEL_CONTAINER_LABEL,
+    CLIPPER_INTERNAL_RPC_PORT, CLIPPER_INTERNAL_QUERY_PORT,
+    CLIPPER_INTERNAL_MANAGEMENT_PORT)
+
+DOCKER_NETWORK_NAME = "clipper_network"
+
+logger = logging.getLogger(__name__)
+
+
+class DockerContainerManager(ContainerManager):
+    def __init__(self,
+                 clipper_public_hostname,
```

Comment:
In retrospect I don't think this comment makes sense. Any docker specific arguments should be provided to init, so that `start_clipper` and `connect` can cleanly inherit from the generic `ContainerManager` interface. I'm leaving this as is.